src/ngAnimate/shared.js

```diff
@@ -144,10 +145,40 @@ function prepareAnimationOptions(options) {
       domOperation = noop;
     };
     options.$$prepared = true;
+
+    if (options.from) {
+      options.from = pixelifyNumericStyles(options.from);
+    }
+
+    if (options.to) {
+      options.to = pixelifyNumericStyles(options.to);
+    }
   }
   return options;
 }
 
+function pixelifyNumericStyles(styles) {
+  for (var prop in styles) {
+    // this should satisfy `z-index` and `zIndex` since
+    // both values are always numeric, but don't require a
+    // `px` prefix.
+    if (prop.charAt(0) === 'z') continue;
```

Comment:
would it be possible to have a list of the properties that do not need the additional `'px'`?
BTW, jQuery list is
```js
cssNumber: {
		"columnCount": true,
		"fillOpacity": true,
		"flexGrow": true,
		"flexShrink": true,
		"fontWeight": true,
		"lineHeight": true,
		"opacity": true,
		"order": true,
		"orphans": true,
		"widows": true,
		"zIndex": true,
		"zoom": true
	}
```